javascript/react_js/router.md

```diff
@@ -5,9 +5,16 @@ In this lecture we are going to learn a little bit about routing in React. Up un
 ### Learning Outcomes
 By the end of this lesson, you should be able to:
 
+- Understand what client-side routing is
 - Explain how `react-router-dom` works
 
-### React Router
+### Client-side Routing
```

Comment:
As per our [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) we want our headings to be [Wikipedia Style Title Case](https://titlecaseconverter.com/rules/#WP):
```suggestion
### Client-Side Routing
```